pkg/controller/podautoscaler/horizontal.go

```diff
@@ -798,10 +807,16 @@ func (a *HorizontalController) reconcileAutoscaler(ctx context.Context, hpaShare
 		setCondition(hpa, autoscalingv2.AbleToScale, v1.ConditionTrue, "SucceededRescale", "the HPA controller was able to update the target scale to %d", desiredReplicas)
 		a.eventRecorder.Eventf(hpa, v1.EventTypeNormal, "SuccessfulRescale", "New size: %d; reason: %s", desiredReplicas, rescaleReason)
 		a.storeScaleEvent(hpa.Spec.Behavior, key, currentReplicas, desiredReplicas)
-		klog.Infof("Successful rescale of %s, old size: %d, new size: %d, reason: %s",
-			hpa.Name, currentReplicas, desiredReplicas, rescaleReason)
+		logger.Info("Successfully rescaled",
+			"hpa", klog.KObj(hpa),
```

Comment:
Let's use "HorizontalPodAutoscaler" instead of "hpa". I'm not sure whether it's common enough that everyone knows what the acronym means - I didn't... :sweat_smile:

Reply:
I'd argue against changing to `HorizontalPodAutoscaler`. IMO it's a well-known abbreviation among pod autoscaling users, e.g. [our docs](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) use it and `kubectl` recognizes it. It's included in most of the logs now (and likely of more logs in the future), so let's use a succinct name.

Reply:
Okay, then let's use "HPA" as acronym. Spelling it entirely with uppercase is consistent with the other cases (like "PVC" for PersistentVolumeClaim") where we use abbreviations.

Reply:
Sure @pohly, @pbetkier. Changes done as suggested.